e2e/packages/test-data/generate-test-data-100.ts

```diff
@@ -0,0 +1,112 @@
+import fs from "node:fs/promises";
+import path from "node:path";
+import { fileURLToPath } from "node:url";
+import { createAnvil } from "@viem/anvil";
+import { execa } from "execa";
+import {
+  ClientConfig,
+  createPublicClient,
+  createWalletClient,
+  encodeEventTopics,
+  http,
+  isHex,
+  numberToHex,
+} from "viem";
+import { mudFoundry } from "@latticexyz/common/chains";
+import { getContract } from "@latticexyz/common";
+import { storeEventsAbi } from "@latticexyz/store";
+import { privateKeyToAccount } from "viem/accounts";
+import IWorldAbi from "../contracts/out/IWorld.sol/IWorld.abi.json";
+
+const LIMIT = 100;
```

Comment:
could this be a parameter in a function instead of a separate script per limit?

Reply:
Ideally, yes! But it'll take some tweaking, the current script writes to one file then kills the whole process (seems this is the only way to shut down `anvil/viem`).

Reply:
Or closes slowly: https://github.com/wevm/anvil.js/issues/41